src/errors/ApiError.js

```diff
@@ -35,22 +35,54 @@
  */
 export default class ApiError {
   /**
-   * @param {String} type - Error type
-   * @param {String} message - Error message
-   * @param {Number} code - Http error code
-   * @param {String|null} trace - Stack trace
+   * @param {AxiosError} error - Axios error
+   * @param {AxiosRequestConfig} error.config - Request config
+   * @param {XMLHttpRequest|ClientRequest} request - Request
+   * @param {AxiosResponse} response - Response
    */
-  constructor(type, message, code, trace = null) {
+  constructor({config, request, response}) {
+    this._config = config;
+    this._response = response;
+    this._request = request;
+
+    this._code = response.status;
+
+    const {type, message, trace} = response.data.error;
```

Comment:
Trace is not in the error anymore, it's in the root.

https://github.com/MapCreatorEU/api/blob/2a5c89b7b86af650ffbfa359d533179fa9984589/app/Macros/ResponseMacros.php#L307

Reply:
I feel like this should be changed back, though